client/test/fixtures/CVRs.txt

```diff
@@ -140,7 +140,7 @@
 {"_precinctId":"21","_ballotStyleId":"12","_ballotId":"PQkmE1LGSSsFClPFGUmrIA==","_testBallot":true,"_scannerId":"scanner-3","_pageNumber":2,"_locales":{"primary":"en-US"},"Justice-1-4":["harriet-tubman"],"Justice-14-2":["marilyn-monroe"],"Justice-14-9":["vincent-van-gogh"],"District-judge-11":["bill-gates"],"District-judge-61":["steven-spielberg"],"District-judge-80":["johan-sebastian-bach"],"District-judge-125":["jane-austen"],"District-judge-127":["angelina-jolie"],"District-judge-129":["salvador-dali"],"district-attorney":["theodore-roosevelt"]}
 {"_precinctId":"20","_ballotStyleId":"5","_ballotId":"yu//ntXFOH5tlXS5xBvqaQ==","_testBallot":true,"_scannerId":"scanner-6","_pageNumber":1,"_locales":{"primary":"en-US"},"president":["jackie-chan"],"representative-district-18":["neil-armstrong"],"lieutenant-governor":["winston-churchill"],"comptroller":["alfred-hitchcock"],"comm-ag":["helen-keller"],"Chief-Justice-1":["stephen-hawking"],"Justice-1-4":["mark-antony"],"Justice-14-2":["marilyn-monroe"],"Justice-14-9":["nikola-tesla"],"District-judge-11":["pablo-picasso"]}
 {"_precinctId":"21","_ballotStyleId":"12","_ballotId":"ZLbMd8RB5TEw44l0RMHJpQ==","_testBallot":true,"_scannerId":"scanner-10","_pageNumber":3,"_locales":{"primary":"en-US"},"county-attorney":["harry-potter"],"sheriff":["robin-williams"],"county-tax-assessor":["christopher-columbus"],"Justice-peace-1-1":["paul-mccartney"],"constable-1":["walt-disney"],"prop-1":["yes"],"prop-2":["no"]}
-{"_precinctId":"23","_ballotStyleId":"12","_ballotId":"E7Rnjqbi8VgRnIOq66Z61w==","_testBallot":true,"_scannerId":"scanner-10","_pageNumber":1,"_locales":{"primary":"en-US"},"president":["tim-allen"],"senator":["earhart"],"representative-district-18":["michael-jordan"],"governor":["thomas-edison"],"lieutenant-governor":["oprah-winfrey"],"ss-justic-3":["john-snow"],"ss-justic-5":["benjamin-franklin"],"comptroller":["andy-warhol"],"comm-ag":["helen-keller"],"Chief-Justice-1":["__write-in"]}
+{"_precinctId":"23","_ballotStyleId":"12","_ballotId":"E7Rnjqbi8VgRnIOq66Z61w==","_testBallot":true,"_scannerId":"scanner-10","_pageNumber":1,"_locales":{"primary":"en-US"},"president":["tim-allen"],"senator":["earhart"],"representative-district-18":["michael-jordan"],"governor":["thomas-edison"],"lieutenant-governor":["oprah-winfrey"],"ss-justic-3":["john-snow"],"ss-justic-5":["benjamin-franklin"],"comptroller":["andy-warhol"],"comm-ag":["helen-keller"],"Chief-Justice-1":["__write-in-0"]}
```

Comment:
Do we want the index here?

Reply:
I think in the long term we don't, but since we are currently producing these types of CVRs, I wanted to test that we can process them.